plugins/osdn/bin/openshift-sdn-docker-setup.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
-
-set -ex
-
-bridge=$1
-mtu=$2
-
-DOCKER_NETWORK_OPTIONS="-b=${bridge} --mtu=${mtu}"
```

Comment:
This causes docker to set the correct MTU on the veths. You need to add that functionality somewhere else if docker isn't going to do it any more.